### DISCUSSION TITLE
Not getting Wyze Verification Code: prompt

### DISCUSSION THREAD

**Discussion by AkoreJepade** [mrlt8/docker-wyze-bridge#1038] [Q&A] [answered]
New to Home Assistant so please be gentle: I have the Add-On installed and it is running with the correct username and password as I am getting the email notification with my WVC (Wyze Verification Code) but when I open the WEB UI I never get the prompt to enter said WVC? 

LOG:

[WyzeBridge] 🏠 Home Assistant Mode

🚀 DOCKER-WYZE-BRIDGE v2.5.0 

 * Serving Flask app 'frontend'
[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
 * Debug mode: off
[WyzeBridge] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.30.33.0:5000
[WyzeBridge] Press CTRL+C to quit
[WyzeBridge] 🔐 MFA Code Required [PrimaryPhone]
[WyzeBridge] 💬 SMS code requested
[WyzeBridge] 📝 Enter verification code in the WebUI or add it to /config/wyze-bridge/mfa_token.txt
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:33] "GET / HTTP/1.1" 200 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:33] "GET /static/site.css HTTP/1.1" 304 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:33] "GET /static/site.js HTTP/1.1" 304 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:33] "GET /api/sse_status HTTP/1.1" 200 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:50] "GET / HTTP/1.1" 200 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:50] "GET /static/site.css HTTP/1.1" 304 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:50] "GET /static/site.js HTTP/1.1" 304 -
[WyzeBridge] 172.30.32.2 - - [06/Nov/2023 14:05:50] "GET /api/sse_status HTTP/1.1" 200 -

**Comment by AkoreJepade** [comment, marked as answer]
Ok, I got it to work. I had to put the ip address of my HA server with :5000 at the end to get the 2FA prompt to show. It took me a few trys and restarts to get it to except it but I think I am good now.

**Reply by KSPSlice** [reply]
Thanks. That worked for me as well!

**Reply by askmahesh1** [reply]
thanks . That worked for me as well.
